src/ripple/app/tx/impl/PayChan.cpp

```diff
@@ -37,86 +37,88 @@ namespace ripple {
+        additional claims. Any transaction that touches the channel after the
+        expiration time will close the channel. When the channel is closed any
+        remaining balance is returned to the owners.
 
     PaymentChannelCreate
 
-        Create a unidirectional channel. The parameters are:
+        Create a bidirectional XRP payment channel. The parameters are:
         Destination
-            The recipient at the end of the channel.
+            The other channel participant.
         Amount
-            The amount of XRP to deposit in the channel immediately.
-        SettleDelay
-            The amount of time everyone but the recipient must wait for a
-            superior claim.
+            The amount of XRP to deposit in the channel from the transaction
+            signer immediately.
         PublicKey
             The key that will sign claims against the channel.
+        DstPublicKey
+            The key that will be used by the other channel participant to sign
+            claims against the channel.
+        SettleDelay
+            The amount of time everyone must wait after a claim for a superior
+            claim.
         CancelAfter (optional)
-            Any channel transaction that touches this channel after the
-            `CancelAfter` time will close it.
-        DestinationTag (optional)
-            Destination tags allow the different accounts inside of a Hosted
-            Wallet to be mapped back onto the Ripple ledger. The destination tag
-            tells the server to which account in the Hosted Wallet the funds are
-            intended to go to. Required if the destination has lsfRequireDestTag
-            set.
-        SourceTag (optional)
-            Source tags allow the different accounts inside of a Hosted Wallet
-            to be mapped back onto the Ripple ledger. Source tags are similar to
-            destination tags but are for the channel owner to identify their own
-            transactions.
+            Do not honor any claims after the `CancelAfter` time if no claims
+            were submitted beforehand.
 
     PaymentChannelFund
 
-        Add additional funds to the payment channel. Only the channel owner may
-        use this transaction. The parameters are:
+        Add additional funds to the payment channel. Either channel participant
+        may use this transaction. The parameters are:
         Channel
             The 256-bit ID of the channel.
         Amount
             The amount of XRP to add.
-        Expiration (optional)
-            Time the channel closes. The transaction will fail if the expiration
-            times does not satisfy the SettleDelay constraints.
 
     PaymentChannelClaim
 
-        Place a claim against an existing channel. The parameters are:
+        Place a claim against an existing channel. A successful claim will
+        cause the channel to close after the settle delay unless a subsequent
```

Comment:
Claims never transfer funds. What do you think about transferring funds after the settle delay, but leaving the channel open (to be closed by submitting a claim tx with no claims or maybe a flag and waiting the settle delay).